infrastructure/Makefile

```diff
@@ -3,6 +3,7 @@
 export ALGORAND_DATA=`pwd`/node_data/PrimaryNode
 export SANDBOX_PATH = ~/.algorand-sandbox
 export REMOTE_REPO = https://github.com/algorand/sandbox.git
+export REACH_PATH = ~/.algo-reach
```

Comment:
how about `algob-reach` or `reach` only ?

Reply:
hmm, i like `.algo-reach`, it resonates with `.algorand-sandbox`. 

`reach` might be weird, as the exec file would be in `~/reach/reach`. Can update to `algob-reach` if you want

Reply:
I think algorand-reach suits better

Reply:
used `algo-reach` because `algorand` is a trademark. but ok.